Sources/Public/AnyCalendarItemModel.swift

```diff
@@ -43,6 +43,8 @@ public protocol AnyCalendarItemModel {
   /// - Note: There is no reason to invoke this function from your feature code; it should only be invoked internally.
   func _isContentEqual(toContentOf other: AnyCalendarItemModel) -> Bool
 
+  mutating func _setSwiftUIWrapperViewContentIDIfNeeded(_ id: AnyHashable)
```

Comment:
kind of annoying that we need this but we're working with a type-erased AnyCalendarItemModel, so we don't know if the original `CalendarItemModel<CalendarItemViewRepresentable>` type is a `SwiftUIWrapperView` or not.